package.json

```diff
@@ -15,7 +15,7 @@
     "./shared-libs/*"
   ],
   "scripts": {
-    "test": "grunt eslint && grunt unit && grunt test-api-integration && grunt e2e",
+    "test": "grunt eslint && grunt unit && grunt test-api-integration",
     "webdriver": "webdriver-manager update && webdriver-manager start",
```

Comment:
When do we use `webdriver`? Does wdio need it? I though it is a protractor thing but I might be very wrong

Reply:
Good catch Jenny ⭐ 
I read a little bit because I had no idea and you are right. I removed the line, thank you!